demos/CMakeLists.txt

```diff
@@ -150,15 +144,9 @@ macro(ie_add_sample)
         if(NOT OpenCV_FOUND)
             message(WARNING "OPENCV is disabled or not found, " ${IE_SAMPLE_NAME} " skipped")
             return()
-        else()
-            add_definitions(-DUSE_OPENCV)
         endif()
     endif()
 
-    if(TARGET IE::ie_cpu_extension)
-        add_definitions(-DWITH_EXTENSIONS)
-    endif()
```

Comment:
It's too early to remove this, as our CI currently uses IE from OpenVINO 2019 R3. It can be removed after the next release (or at least after the next CI update).

Reply:
When do you plan next CI update? If it's planned for a later time then I will roll back it. Please note that this PR should be merged asap because otherwise you may get the build failed (if OpenCV_DIR is not specified explicitly). In last DLDT solution it's implemented that OpenCV_DIR is propagated by `find_package(InferenceEngine)` call to use exactly the same OpenCV both for the DLDT and Demos build.

Reply:
> When do you plan next CI update?

I haven't planned it yet. I usually do it when there's a cause, e.g. a new model that can't be converted by the old version of MO.

Either way, I don't think we need to rush this particular change. I specifically requested this code to be written in this way so that it would work regardless of whether the CPU extension library exists.

> Please note that this PR should be merged asap because otherwise you may get the build failed (if OpenCV_DIR is not specified explicitly).

In our CI we _do_ specify `OpenCV_DIR` explicitly, though.